### PR TITLE
Adding changelog

### PR DESCRIPTION
**Pull request by odscrachel**
Added a changelog file to docs/about

### CODE REVIEW

File: docs/about/changelog.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Changelog
+
```
Comment (duncandewhurst) [inline docs/about/changelog.md:2]:
```suggestion

This page lists changes to the Risk Data Library Standard.

```

File: docs/about/changelog.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+## [X.X.X] - YYYY-MM-DD
+
+The Risk Data Library Standard adheres to [Semantic Versioning](https://semver.org/).
```
Comment (duncandewhurst) [inline docs/about/changelog.md:5]: To my knowledge, semantic versioning isn't yet implemented so I suggest that we omit this for now.

```suggestion
```

File: docs/about/changelog.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+### Schema
+
+### Codelists
+
+### Documentation
+
```
Comment (duncandewhurst) [inline docs/about/changelog.md:12]:
```suggestion
### Normative documentation

### Non-normative documentation

```

**Review by duncandewhurst — CHANGES_REQUESTED** [review]

**Comment by duncandewhurst** [comment]
@odscjen, since Rachel is on leave, I applied the suggestions from my review and resolved the check failure. Please could you review.

**Review by odscjen — DISMISSED** [review]

**Comment by odscjen** [comment]
passing back to you @duncandewhurst as it's still having merging blocked due to your requested changes (even though you've applied them)

**Review by duncandewhurst — APPROVED** [review]

**Comment by duncandewhurst** [comment]
I realised that `changelog.md` was not listed in any toctree so I've added it under `about/history`. I've also configured ReadtheDocs to fail on warnings for missing references and documents that aren't included in any toctree. Those issues would have been caught by running `make autobuild` locally so it's good to have ReadtheDocs follow the same behaviour. I think this is now ready for review by @stufraser1 or @matamadio.

**Review by stufraser1 — APPROVED** [review]
I focussed on the structure of changelog, which looks good.

Note we've made lots of change to docs on branch docs.mat compared to dev - we obviously need to merge these changes with those.

**Pull request merged**